src/sms/tokens.ts

```diff
@@ -1,5 +1,6 @@
 const { sparqlTokenMap, stardogSparqlTokens } = require('../sparql/tokens');
-import { TokenType, createToken, IToken } from 'chevrotain';
+import { TokenType, IToken } from 'chevrotain';
+import { createKeyword } from '../sparql/keywords';
```

Comment:
not sure if it matters here, but this might also be a place where we need to use CommonJS syntax: https://github.com/SAP/chevrotain/issues/345

Reply:
On second thought, it might be better to import this directly from the helpers instead of through the sparql keywords file

Reply:
will sms always follow the same `longer_alt` as sparql? 

The helper has a overridable token/keyword and sparql has a token/keyword that defaults `longer_alt` to unknown. This is an intended distinction so any language can create their own flavor of a token/keyword.

This can be useful unless we need all languages to default to unknown

Reply:
I suppose this is SPARQL specific. We want SMS to have the same behavior for keywords as the SPARQL keywords since SMS is extended from SPARQL. In that case it would make sense to keep `createKeyword` defined in the SPARQL keywords.